flowzone.yml

```diff
@@ -1617,12 +1620,20 @@ jobs:
           node-version: 18
 
       - name: Docusaurus Builder
-        uses: product-os/docusaurus-builder@master
+        if: |
+          needs.project_types.outputs.has_readme == 'true' &&
+          inputs.cloudflare_website != '' && inputs.custom_website_build == ''
+        uses: product-os/docusaurus-theme@master
         with:
           repo: ${{ github.event.repository.name }}
           org: ${{ github.repository_owner }}
           default_branch: ${{ github.event.repository.default_branch }}
-          url: https://${{ inputs.cloudflare_website }}.pages.dev
+          url: https://${{ inputs.cloudflare_website }}.pages.dev/
```

Comment:
@vipulgupta2048 we are still limited to `.pages.dev` here. What if a user wants `https://flowzone.foo.bar`?

Reply:
@anujdeshpande is will be opening a PR about it very soon. Already planned and discussed.